Clean_Data.R

```diff
@@ -0,0 +1,128 @@
+#####################################################################################################################
+#####   Assignment: Replication Project (Part 1)                                                                #####
+#####   Due Date:   3/15/2015                                                                                   #####
+#####   Authors:    Megan Blanchard and Kalyani Jayasankar                                                      #####
+#####   Input:      Raw data file(1626360926english_merge_2010_americasbarometer_v14v3.dta)                     ##### 
+#####   Output:     Cleaned data files for analysis (clean2010data.dta, colorr_recode_subset.dta)               #####
+#####   Notes:      This is the English merged 2010 data downloaded from http://www.americasbarometer.org       #####
+#####               This file reads in the raw data and preps for analysis. In this file we:                    ##### 
+#####                      1-subset to 23/26 total countries                                                    #####
+#####                      2-create 4 new variables (tone, region,colorr_recode & parent_occ) for anaylsis      #####
+#####################################################################################################################
+
+#install.packages("foreign", "broom")
+
+library(foreign)
+library(dplyr)
+library(ggplot2)
+library(broom)
+
+#getwd()
+setwd("~/Desktop/R working directory/RepProj")
```

Comment:
try to avoid global path names; this won't work on a different machine or if you change you directory structure